services-directions/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -192,6 +192,114 @@ public void directionsRequestResult_doesContainOverview() throws Exception {
     assertEquals(DirectionsCriteria.OVERVIEW_SIMPLIFIED, route.routeOptions().overview());
   }
 
+  @Test
+  public void directionsRequestResult_doesContainBicycleType() throws Exception {
+    Response<DirectionsResponse> response = MapboxDirections.builder()
+      .geometries(DirectionsCriteria.GEOMETRY_POLYLINE6)
+      .baseUrl(mockUrl.toString())
+      .accessToken(ACCESS_TOKEN)
+      .origin(Point.fromLngLat(1.0, 1.0))
+      .destination(Point.fromLngLat(5.0, 5.0))
+      .profile(DirectionsCriteria.PROFILE_WALKING)
+      .continueStraight(false)
+      .language(Locale.CANADA)
+      .bicycleType("Road")
+      .alternatives(true).build().executeCall();
+    DirectionsRoute route = response.body().routes().get(0);
+
+    assertEquals("Road", route.routeOptions().bicycleType());
+  }
+
+  @Test
+  public void directionsRequestResult_cyclingSpeed() throws Exception {
```

Comment:
This is checking the same as `directionsRequestResult_doesContainOverview`. I guess because this PR is still a WIP.

The same thing happens with `directionsRequestResult_doesContainUseRoads`, `directionsRequestResult_doesContainUseHills`, `directionsRequestResult_doesContainUseFerry` and `directionsRequestResult_doesContainAvoidBadSurfaces` tests.